docs/other/configuration-with-osg-configure.md

```diff
@@ -0,0 +1,550 @@
+as well as the [Overview of Services](OverviewOfServicesInOSG) and [Site Planning](SitePlanning) discussions
+for explanations of the various storage models and the requirements for them.
+
+This section is contained in `/etc/osg/config.d/10-storage.ini` which is provided by the `osg-configure-ce` RPM.
+
+| Option           | Values Accepted | Explanation                                                                                                                                                                                                    |
+|------------------|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **se_available** | `True`, `False` | This indicates whether there is an associated SE available.                                                                                                                                                    |
+| default_se       | String          | If an SE is available at your cluster, set default_se to the hostname of this SE, otherwise set default_se to UNAVAILABLE.                                                                                     |
+| **grid_dir**     | String          | This setting should point to the directory which holds the files from the OSG worker node package. See note                                                                                                    |
+| **app_dir**      | String          | This setting should point to the directory which contains the VO specific applications. See note                                                                                                               |
+| data_dir         | String          | This setting should point to a directory that can be used to store and stage data in and out of the cluster. See note                                                                                          |
+| worker_node_temp | String          | This directory should point to a directory that can be used as scratch space on compute nodes. If not set, the default is UNAVAILABLE. See note                                                                |
+| site_read        | String          | This setting should be the location or url to a directory that can be read to stage in data via the variable `$OSG_SITE_READ`. This is an url if you are using a SE. If not set, the default is UNAVAILABLE    |
+| site_write       | String          | This setting should be the location or url to a directory that can be write to stage out data via the variable `$OSG_SITE_WRITE`. This is an url if you are using a SE. If not set, the default is UNAVAILABLE |
+
+
+!!! note
+    All of these can be defined in terms of an environment variable
+    (e.g. `$FOO`) that will be evaluated on the worker node.
+
```

Comment:
Did you really want the text that follows this to all be in the note?

Reply:
derp derp